test/e2e/tests/inputs/text.spec.ts

```diff
@@ -78,20 +78,25 @@ test.describe('inputs: text', () => {
     await createDraftDocument('/test/content/book')
 
     const titleInput = page.getByTestId('field-title').getByTestId('string-input')
-    const paneFooter = page.getByTestId('pane-footer')
+    const paneFooter = page.getByTestId('pane-footer-document-status')
     const publishButton = page.getByTestId('action-Publish')
 
     await titleInput.fill('Title A')
 
+    // generally waiting for timeouts is not a good idea but for this specific instance
+    // since we are using `.fill` and `.click` they can cause the draft creation and publish to happen at the same exact time.
+    // We are waiting for 1s to make sure the draft actually gets created and click action is not too eager
+    await page.waitForTimeout(1000)
+
     // Wait for the document to be published.
     publishButton.click()
-    await expect(paneFooter).toContainText('Published just now')
+    expect(await paneFooter.textContent()).toMatch(/published/i)
```

Comment:
using regex here to avoid more flake, the publish can take time and it will respond with published 1 min ago, etc. This just looked for the word published instead of exactly "just now"